packages/sources/src/components/card/card.args.ts

```diff
@@ -29,9 +35,18 @@ export const cardArgTypes: ArgTypes<Card> = {
   }
 };
 
-export function cardArgsMapper(a: CardArgs): Card {
+export function cardArgsMapper(a: CardArgs): Card<never> {
   return {
     label: a.label,
+    selectable: a.selectable
+      ? {
+        id: '1',
```

Comment:
dat mag toch niet, id's in HTML hebben toch een strenge naming convention?

Reply:
https://www.w3.org/TR/2011/WD-html5-20110525/elements.html#the-id-attribute
Toch wel dus.